src/components/AssignmentList.js

```diff
@@ -9,9 +9,10 @@ const AssignmentList = ({ courseId, assignments, isAssistant, onAssignmentExport
             const label = `Assignment ${index + 1} - ${assignment.title}`;
             assignment.label = label;
             return (
-                <li className="list-group-item" key={assignment.id}>
+                <li key={assignment.id}>
                     <Link to={`/courses/${courseId}/assignments/${assignment.id}`} key={assignment.id}>
-                        <p className="h6">{label}</p>
+                        <strong>{label}</strong>
+                        <br />
                         <small>{Util.timeFormatter(assignment.dueDate)}</small>
                     </Link>
                     {isAssistant &&
```

Comment:
If you log in with `ludwig.abeille@uzh.ch:test` you will see extra buttons here and a small table to export student results.
Could you maybe work your magic on it as well? :D

Reply:
will do